components/tyk-gateway/templates/deployment-gw-repset.yaml

```diff
@@ -115,6 +115,8 @@ spec:
             value: "{{ .Values.global.oasValidateExamples }}"
           - name: TYK_GW_OAS_VALIDATE_SCHEMA_DEFAULTS
             value: "{{ .Values.global.oasValidateSchemaDefaults }}"
+          - name: TYK_GW_ENABLE_FIXED_WINDOW_RATE_LIMITER
+            value: "{{ .Values.gateway.enableFixedWindowRateLimiter }}"
```

Comment:
```suggestion
            value: "{{ .Values.gateway.enableFixedWindowRateLimiter | quote }}"
```

Reply:
I'm not sure what this is supposed to do, seems redundant?

- boolean value
- already has quotes outside the template tag
- previous RL PR #238 also did not have quote

Reply:
Ohh sorry. Yes It's redundant.